addons/message-filters/addon.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Message filtering",
+  "description": "Enables the ability to finely filter messages on the messages page through checkboxes.",
```

Comment:
Addon name and description should make it clear that 1. addon works in messaging page, does not affect other addons that access messages 2. does not exclude the messages you unselect from the message count. I think the best way to do that is to use the words "see" or "hide" (to imply the message count won't be affected) and explicitly say "from the messages page"